pkg/ccl/streamingccl/streamproducer/replication_stream_test.go

```diff
@@ -375,13 +378,8 @@ USE d;
 		})
 		require.Nil(t, err)
 
-		expected := replicationtestutils.EncodeKV(t, srcTenant.Codec, t2Descr, 42)
-		feed.ObserveKey(ctx, expected.Key)
 		feed.ObserveError(ctx, func(err error) bool {
-			return strings.Contains(err.Error(), "unexpected MVCC history mutation") ||
-				// TODO(casper): disabled this once we figured out why we have context cancellation
-				// emitted from ingestion side.
-				strings.Contains(err.Error(), "context canceled")
+			return strings.Contains(err.Error(), "unexpected MVCC history mutation")
```

Comment:
I assume we have run this under stress and haven't see context canceled any longer?